Shared/PostList/PostListView.swift

```diff
@@ -133,7 +133,7 @@ struct PostListView: View {
                 }
             }
         }
-        .ignoresSafeArea()
+        .ignoresSafeArea(.all, edges: .bottom)
```

Comment:
@thebaer This is really the only line that matters to this PR — the rest is cleanup from deleting the no-longer-needed UINavigationController+Appearance file.